test/functional/kaui/bundles_controller_test.rb

```diff
@@ -82,6 +82,32 @@ class Kaui::BundlesControllerTest < Kaui::FunctionalTestHelper
     assert_redirected_to account_bundles_path(@bundle.account_id)
   end
 
+  test 'should get pause_resume ' do
+    get :pause_resume, :id => @bundle.bundle_id
```

Comment:
FYI, pausing/resuming a subscription bundle may impact the invoicing (in some cases, the paused period won't be billed). This could have side effects on other tests if we keep reusing the same bundle over and over for various checks (might be worth creating a new one in some cases like that).